scripts/config.pl

```diff
@@ -87,6 +87,7 @@
 MBEDTLS_DEPRECATED_REMOVED
 MBEDTLS_HAVE_SSE2
 MBEDTLS_PLATFORM_NO_STD_FUNCTIONS
+MBEDTLS_CTR_DRBG_USE_128_BIT_KEY
```

Comment:
If this should be tested separately, then shouldn't we add a build for it in all.sh?

Reply:
Yes, but I wanted to stop increasing this PR's scope. The new build is in https://github.com/ARMmbed/mbed-crypto/pull/291 (I'll make backports once it's approved).